server/honeybadger/processors.py

```diff
@@ -100,12 +102,16 @@ def process_wlan_survey(data):
 
 def process_ip(data):
     logger.info('Processing IP address.')
-    coords = get_coords_from_uniapple(data['ip'])
+    coords = get_coords_from_ipstack(data['ip'])
+    if not coords:
+        logger.info('Using fallback API.')
+        coords = get_coords_from_ipinfo(data['ip'])
+
     if all([x for x in coords.values()]):
         add_beacon(
             target_guid=data['target'],
             agent=data['agent'],
-            ip=data['ip'],
+            ip=data['ip'],  # Replace with get_external_ip(data['ip'])?
```

Comment:
This was the line that didn't make sense, right?

Reply:
Correct. This comment can be removed. Any external IP translation should be performed by the agent before calling the API.